Server.Logic/Managers/RecipeManager.cs

```diff
@@ -320,6 +330,21 @@ await _dbInfo.RecipiesImages.AddAsync(new RecipiesImage()
         return new ManagerActionResult<string>(path, ResultEnum.OK);
     }
 
+    public async Task<IManagerActionResult<RecipeExtendedViewModel>> GetRecipeDetailsAsync(int recipeId, string user)
+    {
+        var recipe = await _dbInfo.Recipes
+            .Include(x => x.RecipiesIngredients)
+            .Include(x => x.RecipiesImages)
+            .Include(x => x.Categories)
+            .FirstOrDefaultAsync(x => x.Id == recipeId);
+
+        if (recipe == null)
+            return new ManagerActionResult<RecipeExtendedViewModel>(null, ResultEnum.BadRequest, _recipeNotFound);
+        
+        var result = _recipeViewModelFactory.CreateExtendedViewModel(recipe, user, (await GetMainImageAsync(recipeId))?.ImagePath);
+        return new ManagerActionResult<RecipeExtendedViewModel>(result, ResultEnum.OK);
```

Comment:
I pytanie otwarte bo widzę że teraz nigdzie nie zwracamy listy użytkowników którzy to lubią, pytanie czy front to będzie chciał

Reply:
zwracamy w getById ( i narazie chyba nie ma żadnych features które tego wymagają)

Reply:
a jak tak to git sorry